.github/workflows/pull_request_trigger_rococo.yml

```diff
@@ -0,0 +1,16 @@
+name: Remote PR Action Initiator Litentry Parachain
+ 
+on:
+  push:
+    branches: rococo
+ 
+jobs:
+  pull-request-trigger-parachain:
+    runs-on: ubuntu-latest
```

Comment:
should we fix the os version, e.g. ubuntu-20.04, since most of our services run on this version.

Reply:
sounds good. we can make it as a to-do point. currently it should be no harm with latest, in my opinion.